src/thread.hpp

```diff
@@ -74,6 +74,8 @@ class thread_t
 
     //  Creates OS thread. 'tfn' is main thread function. It'll be passed
     //  'arg' as an argument.
+    //  Name is 16 characters max including terminating NUL. Thread naming is
+    //  implemented only for pthread and windows when a debugger is attached.
```

Comment:
I suggest a comma here otherwise it's misleading: "only for pthread, and windows when a debugger is attached"